framework/wazuh/syscollector.py

```diff
@@ -162,6 +250,28 @@ def get_netiface_agent(agent_id, offset=0, limit=common.database_limit, select={
                           nested=nested, query=q)
 
 
+def get_hotfixes_agent(agent_id, offset=0, limit=common.database_limit, select={}, search={}, sort={}, filters={}, nested=True):
```

Comment:
q param is missing in this function. It must be added to be coherent with the rest of the functions in this module.